src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeController.java

```diff
@@ -32,9 +33,37 @@ public class CourseOverTimeController {
     @ApiOperation(value = "Get a list of courses over time")
     @GetMapping(value = "/search", produces = "application/json")
     public ResponseEntity<String> search(
+        @ApiParam(
+            name = "StartQtr",
+            type = "string",
+            value = "starting quarter in yyyyq format, e.g. 20231 for W23, 20232 for S23, etc. (1=Winter, 2=Spring, 3=Summer, 4=Fall)",
+            example = "20231",
+            required = true
+        )
         @RequestParam String startQtr,
+        @ApiParam(
+            name =  "endQtr",
+            type = "String",
+            value = "ending quarter in yyyyq format, e.g. 20231 for W23, 20232 for S23, etc. (1=Winter, 2=Spring, 3=Summer, 4=Fall)",
+            example = "20231",
+            required = true
+        )
         @RequestParam String endQtr,
+        @ApiParam(
+            name =  "subjectArea",
+            type = "String",
+            value = "simplified area name followed with full area name",
+            example = "EARTH - Earth Science",
+            required = true
+        )
         @RequestParam String subjectArea,
+        @ApiParam(
+            name =  "courseNumber",
+            type = "String",
+            value = "the specific course number",
+            example = "16 or 130A (for CS16 or CS130A)",
```

Comment:
It might be better to have this as a specific example, since the field is automatically populated when trying it out in Swagger. Like just "130A".